### ISSUE TITLE
[BRUKERHISTORIE] Navigere fra en tjeneste til en informasjonsmodell

### ISSUE THREAD

**Issue by KurtStian** [Informasjonsforvaltning/behov#535]
Som: konsument av en tjeneste
Ønsker jeg å: se hvilke informasjonsmodeller som er input (evidence) og output til en tjeneste og navigere til disse
Slik at: jeg kan se hvilke data som inngår i en tjeneste

Relatert til epos: #436 

Lenke til design: 
Lenke til løsningsarkitektur:

Forretningsverdi (Kritisk/Viktig/Mindre viktig/Kjekt å ha): Viktig

**Comment by KurtStian** [comment]
Per i dag er denne koblingen ikke implementert i CPSV-profilen.

**Comment by lillelars** [comment]
@KurtStian @stigbd har vi konkludert med noe her? Skal vi benytte evidence til å lenke tjenester mot informasjonsmodeller?

**Comment by KurtStian** [comment]
Avventer behovet da det ikke finnes støtte for dette i cpsv-ap-no. Det kan være aktuelt å se nærmere på behovet hvis det skulle bli etterspurt fra konkrete miljø.

**Comment by KjerstiSteien** [comment]
@KurtStian @BRespen kan vi lukke denne og heller opprette ny sak dersom noen spiller inn dette behovet senere?

**Comment by KurtStian** [comment]
@KjerstiSteien @BRespen Ja, jeg tenker at denne kan lukkes da spesifiksjonen ikke støtter dette behovet.
Lukker saken.
